docs/openapi3.yaml

```diff
@@ -2423,6 +2423,9 @@ components:
         events:
           description: list of events which will trigger the webhook
           type: array
+          uniqueItems: true
+          minItems: 1
+          maxItems: 7
```

Comment:
i wouldn't include this in the spec since if we add another `webhook_type` we will need to remember to increase this parameter,

a better condition would be to check that the webhook type fits the enum stated in the spec + uniqueItems as you wrote -> meaning we will never have more webhook_types than what we specified if those two conditions are met

Reply:
removed maxItems restriction